src/main/java/com/epam/prd/jjop/helpers/com/epam/map/MapOfWithImutableClass.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.prd.jjop.helpers.com.epam.map;
+
+import java.util.Map;
+
+/**
+ * immutable Map with custom immutable class

```

Comment:
Make this a proper English sentence. Javadocs should be written in the fully correct language. Starting from an upper case letter, ends with `.`, `?`, or `!`.